README.md

```diff
@@ -1,3 +1,189 @@
 # CNV-patissier
 
-Orchestrates your CNV bakeoff, part of research project for Clinical Bioinformatics Scientist Training Programme. 
+Orchestrates your Copy Number Variant (CNV) bakeoff, part of research project for Clinical Bioinformatics Scientist Training Programme. 
+
+## Overview
+
+This project is aimed to collect data on the ability of CNV callers (targeted-capture Next Generation Sequencing) to detect clinically relevant CNVs. All samples should have a gold-standard known-CNV status for the gene in question. This project is aimed at bioinformaticians working genetics laboratories but please let me know via the issues if there are any problems in installation or running, regardless. 
```

Comment:
I think you're missing an "in".

Should this be:
"aimed at bioinformaticians working in genetics laboratories"

Reply:
:+1: